src/main/kotlin/org/jitsi/videobridge/transport/octo/BridgeOctoTransport.kt

```diff
@@ -274,7 +274,7 @@ class BridgeOctoTransport(
         }
 
         fun largePacketSent(mediaType: MediaType) {
-            val value = largePacketsSent.getValue(mediaType).incrementAndGet()
+            val value = largePacketsSent.computeIfAbsent(mediaType) { AtomicLong() }.incrementAndGet()
```

Comment:
Uh oh, why didn't the `withDefault` work? (arguably should be a different PR)

Reply:
`withDefault` returns a default value without storing it in the map (i.e. returns a new AtomicLong() every time). I'll open a separate PR.

Reply:
🤦  Ugh, ok that was my bad.  I can do the other PR if you want.  Let me know.